op-service/client/no_op_bss_client.go

```diff
@@ -1,33 +1 @@
 package client
```

Comment:
Can we delete this file instead of leaving it empty?

Reply:
yup 👍